build-tools/packages/version-tools/src/bumpTypes.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License.
  */
 
+import type { Range, SemVer } from "semver";
 import * as semver from "semver";
```

Comment:
Since `semver` is wholly imported. Why not use `semver.Range` and `semver.SemVer` for full consistency?

Reply:
Intellisense fixing FTW 😎 Will clean up.